azure-pipelines.yml

```diff
@@ -174,18 +174,38 @@ jobs:
           trigger_sha: '$(trigger_sha)'
       - template: ci/report-end.yml
 
-  - job: compatibility
+  - job: compatibility_linux
     dependsOn:
       - check_for_release
     timeoutInMinutes: 60
     pool:
-      name: 'linux-pool'
+      name: linux-pool
+    steps:
+      - template: ci/report-start.yml
+      - checkout: self
+      - template: ci/compatibility.yml
```

Comment:
Let’s leave it like this for now but I wonder if it might make sense to separate building the platform-agnostic artifacts from building the SDK tarball and running the tests. That might make the logic a bit cleaner and we can start running the macos job sooner.